Fix speechbubble

Hout opened the pull request:
Enabled arrows on left and right sides. Disabled arrows to be placed on corners; that looks ugly.

Three20 was unable to position the arrows on the speechbubbles on the side edges. This is corrected now.
In addition, positioning the arrow at 45 degrees would overlap the rounded corner. The arrow will now be placed right next to the rounded corner.

This has been mentioned on the forums in the past but proper code was never submitted.

Hout committed:
Enabled arrows on left and right sides. Disabled arrows to be placed on corners; that looks ugly.

jwang commented:
Thanks. can you provide a simple test case using the TTCatalog source as an example to verify?

Hout committed:
Fixed a bug with pointers on the left hand edge.
Included samples for pointers in various formats in TTCatalog.
Added some documentation in TTSpeechBubbleShape.h.

Hout commented:
Done and it was a good idea as I identified a bug and fixed it ;-)
I included a few speech bubbles in TTCatalog with various pointers.

2011/5/29 jwang <
reply@reply.github.com>

> Thanks. can you provide a simple test case using the TTCatalog source as an
> example to verify?
> 
> ## 
> 
> Reply to this email directly or view it on GitHub:
> https://github.com/facebook/three20/pull/556#issuecomment-1256416
